WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift

```diff
@@ -52,6 +52,11 @@ final class ProductSelectorViewModel: ObservableObject {
     ///
     @Published private(set) var productRows: [ProductRowViewModel] = []
 
+    /// Determines if products should be toggled on selection
+    /// `false` by default, mainly used to allow multi-seleciton on the Order Creation flow
+    ///
+    let shouldToggleItemOnSelection: Bool
```

Comment:
I am trying to understand it here 😅 What does it mean toggle on selection, and why would we want that they are not toggled?

Reply:
> I am trying to understand it here 😅 What does it mean toggle on selection, and why would we want that they are not toggled?

That's a good question, I'm trying to come up with a proper name for this flag, for the moment has been changed to `supportsMultipleSelections` in the latest commits ([Convo here](https://github.com/woocommerce/woocommerce-ios/pull/8935#discussion_r1117872002)). 

The flag is used to determine if we allow items to be multi-selected, and logic happens when we select them. This is the main difference between the Order and the Coupons implementation, the latter allows product multi-selection but nothing is triggered while products are selected (just UI), while in the Order implementation we trigger a callback each time a product is selected, so we can add/remove them to the Order.